scripts/obtain/uninstall/dotnet-uninstall-pkgs.sh

```diff
@@ -31,8 +34,11 @@ remove_dotnet_pkgs
 [ "$?" -ne 0 ] && echo "Failed to remove dotnet packages." && exit 1
 
 echo "Deleting install root - $dotnet_install_root"
-rm -rf $dotnet_install_location
+rm -rf $dotnet_install_root
 rm -f $dotnet_path_file
 
+echo "Deleting NuGet package location"
```

Comment:
@Sridhar-MS I thought that there is only `$HOME/.nuget`? Why do we have these? 

@brthor what do you mean? This is impacting also built shared apps, since you can run your built output out of the NuGet cache.